llvm/lib/SYCLLowerIR/SYCLVirtualFunctionsAnalysis.cpp

```diff
@@ -0,0 +1,117 @@
+//===--- SYCLVirtualFunctionsAnalysis.cpp ---------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file contains implementation of SYCLVirtualFunctionsAnalysis pass that
```

Comment:
```suggestion
// This file contains the implementation of the SYCLVirtualFunctionsAnalysis pass that
```

Reply:
Applied in 57e43cf0fd010cd160fc9ebe9073b7c7c2711cc9